chromium_src/third_party/blink/renderer/core/execution_context/navigator_base.cc

```diff
@@ -5,15 +5,66 @@
 
 #include "third_party/blink/renderer/core/execution_context/navigator_base.h"
 
+#include "base/compiler_specific.h"
+#include "base/system/sys_info.h"
+#include "third_party/abseil-cpp/absl/random/random.h"
 #include "third_party/blink/public/platform/web_content_settings_client.h"
+#include "third_party/blink/renderer/core/execution_context/execution_context.h"
 #include "third_party/blink/renderer/core/frame/local_dom_window.h"
 #include "third_party/blink/renderer/core/frame/local_frame.h"
 #include "third_party/blink/renderer/core/loader/frame_loader.h"
+#include "third_party/blink/renderer/core/probe/core_probes.h"
+
+namespace blink {
+namespace probe {
+
+const unsigned kFakeMinProcessors = 2;
+const unsigned kFakeMaxProcessors = 8;
```

Comment:
should be either static/in anonymous ns or located right in the function.

Reply:
Moved to the function in [eedc83c](https://github.com/brave/brave-core/pull/13498/commits/eedc83c44654f38ebfaf1e60ac1d018e5b8e2ca3)